src/sssom/util.py

```diff
@@ -1118,20 +1118,20 @@ def _is_iri(string: str) -> bool:
 def get_prefix_from_curie(curie: str) -> str:
     """Get the prefix from a CURIE."""
     if _is_curie(curie):
-        return curie.split(":")[0]
+        return ReferenceTuple.from_curie(curie).prefix
     else:
         return ""
 
 
-def get_prefixes_used_in_table(df: pd.DataFrame, converter: Converter) -> Set[str]:
+def get_prefixes_used_in_table(df: pd.DataFrame) -> Set[str]:
     """Get a list of prefixes used in CURIEs in key feature columns in a dataframe."""
     prefixes = set(SSSOM_BUILT_IN_PREFIXES)
     if df.empty:
         return prefixes
     sssom_schema_object = _get_sssom_schema_object()
     entity_reference_slots = sssom_schema_object.entity_reference_slots & set(df.columns)
     new_prefixes = {
-        converter.parse_curie(row).prefix
+        ReferenceTuple.from_curie(row).prefix
```

Comment:
Does this change the behaviour of the method slightly?

Did `converter.parse_curie` obtain a prefix _only_ of the specific converter actually had that prefix?

If so, the method `get_prefixes_used_in_table` was previously returning "get_valid_prefixes_used_in_table_according_to_prefix_map", right?

I am not objecting to the change, just wanted to make sure this was indeed happening.

Reply:
the old implementation of `curies.Converter.parse_curie` just deferred to `        ReferenceTuple.from_curie(row).prefix`. Now, the implementation validates against the prefix map. So this change maintains current SSSOM functionality, where the goal was to find _all_ prefixes, not just valid ones

Reply:
Thank you!